example/styles.css

```diff
@@ -160,9 +160,6 @@ button:not(.cf-btn) {
 }
 
 .cf-card__footer_message {
```

Comment:
We should be able to delete all the custom css you added right?  `<Text/>` css should already be handled from when the examples for that component were created.